File: src/difficulty_1/difficulty_1_3/nastyHacks/NastyHacks.java
package difficulty_1.difficulty_1_3.nastyHacks;

import java.util.Scanner;

public class NastyHacks {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int numberOfCases = scanner.nextInt();

        for(int i = 0; i < numberOfCases; i++) {
            // the expected revenue if you do not advertise
            long noAdvertise = scanner.nextLong();
            // the expected revenue if you do advertise
            long withAdvertise = scanner.nextLong();
            // the cost of advertising
            long costOfAdvertise = scanner.nextLong();

            long cost = withAdvertise - costOfAdvertise;
            if(noAdvertise < cost) {
                System.out.println("advertise");
            } else if(noAdvertise > cost){
                System.out.println("do not advertise");
            } else {
                System.out.println("does not matter");
            }
        }
    }
}

File: KotlinImplementation/src/difficulty_1/NastyHacks.kt
package difficulty_1

fun main() {
    val n = readLine()!!.toInt()
    val builder = StringBuilder()

    for(i in 1..n) {
        val (r, e, c) = readLine()!!.split(' ').map(String::toInt)

        val result = e - c

        if(r == result)
            builder.append("does not matter").append("\n")
        else
            if (r > result)
                builder.append("do not advertise").append("\n")
            else
                builder.append("advertise").append("\n")
    }

    println(builder)
}